.changeset/fifty-pigs-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': patch
+---
+
+cluster.caFile configuration is used when the Middleware is created on KubernetesProxy
```

Comment:
The project automation will turn this into a release note in a changelog entry like [this](https://github.com/backstage/backstage/blob/master/docs/releases/v1.17.0-changelog.md#backstageplugin-kubernetes-backend0113), so it doesn't have to be so technical/implementation-heavy; it's more relevant to highlight what integrators need to know when they upgrade. Maybe something like
```suggestion
Fixed a bug where requests to the proxy endpoint would fail for clusters with `caFile` configured
```